site_scons/site_tools/extra/extra.py

```diff
@@ -53,6 +57,10 @@ def _supports_correct_style(clang_exe):
     except subprocess.CalledProcessError:
         return False
 
+    match = re.search(r"google3\-trunk", output)
+    if match:
```

Comment:
Since it adds a pylint warning, it's probably not what we want anyway.  Is there an alternative way to check this?  Perhaps using a simple example  that 14.0.5 fixed and checking diff?

Reply:
I know you've removed this but what are you attempting here?  Are you using a custom llvm version?  This code is executed from both the build (scons) and the commit hook so I don't think we want to be doing complex configure style checks.

Reply:
yeah, it was a build of clang-format.  I talked to my boss and he said we could just set it up to use a valid, normal version